modules/contact/doc/content/source/constraints/MechanicalContactConstraint.md

```diff
@@ -51,6 +51,9 @@ Note that the `kinematic` method uses penalty parameter only to enforce the cond
 
 For the frictional contact (`coulomb` model), the `penalty` method calculates normal force and tangential predictor force using penalty stiffness and check for frictional capacity. While using the `kinematic` method, the deformation of surrounding material serves the role of the penalty parameter.
 
+### Contact gap offset
+
+Gap offset can be provided to the current mechanical contact constraint. It can be either `secondary_gap_offset` (gap offset from secondary side) or `mapped_primary_gap_offset` (gap offset from primary side but mapped to secondary side). This could be useful to model contact in such as plates or shells. However, the offsetted gap is treated as rigid region without deformation.
```

Comment:
I'd replace the last two sentences with this (I'd drop the part about plates or shells because although that's correct, we don't support that yet, so I wouldn't want to give the impression that we do):

Use of these gap offset parameters treats the surfaces as if they were virtually extended by the specified amount, so that the surfaces are treated as if they are closer than they actually are. There is no deformation within the material in this gap offset region.